### ISSUE TITLE
Which .py file contains the main algorithm in your AAAI paper?

### ISSUE THREAD

**Issue by Newbeeer** [Sunarker/Safeguarded-Dynamic-Label-Regression-for-Noisy-Supervision#1]
I wonder about which file contains the LCCN training codes? Is it cifar10_train_varC.py?

Thanks

**Comment by Sunarker** [comment]
Oh, yes. I will enrich the readme.

**Comment by Newbeeer** [comment]
Sorry to bother. I have some problems re-implemented your method in Pytorch.
How do you pre-train your ResNet32?

**Comment by Newbeeer** [comment]
Also, what's the data loaded for trans_warming ? train_init or eye(10)?

**Comment by Sunarker** [comment]
Hi, I have updated the Readme. It can answer your question about the pre-training of ResNet32. For the trans_warming, we use the initialization mentioned in [S-adaptation](https://openreview.net/references/pdf?id=Sk5qglwSl), i.e., the noise transition estimated by the M-step. It is pre-computed by our cifar10_train_T.py as a byproduct and saved in one pickle file.

**Comment by Newbeeer** [comment]
Hi, do you train the model for 120 epoch, and follow the 0.5-0.1-0.01 learning rate setting in pre-training?

**Comment by Sunarker** [comment]
I use the model trained by CE to initialize the Forward, S-adaptation and LCCN models. This setting is indicated by the the argument init_dir as shown in the training command. So about your question, you can check the training procedure of CE.

**Comment by Newbeeer** [comment]
I see. Thank you
